### ISSUE TITLE
zombie attack bug

### ISSUE THREAD

**Issue by Real-Eragamer** [TheGameCreators/GameGuruRepo#5687]
https://github.com/user-attachments/assets/c816c4a2-65f0-4f26-b400-f01759e7ee58

i'm having problem with zombie attack lua script. near 2 month ago i was curious and i changed some of it on ai management and since that as you can see i'm having this problem. i tried removing zombie attack lua script and verfy software on steam to redownload it but this issue still remains. i'll be thanksful if you help me about it.

**Comment by synchromesh62** [comment]
@Real-Eragamer 
Hmm just tested on a blank map and all good here ?
I assume you have tried verifying etc.

https://github.com/user-attachments/assets/58f9a0c5-95a2-4f54-bf07-b16385224e06

**Comment by Real-Eragamer** [comment]
> @Real-Eragamer Hmm just tested on a blank map and all good here ? I assume you have tried verifying etc.
> 
>  Z1.mp4

yeah i did verfing and the script was downloaded again but the issue still remains.

as i mentioned , i was curious and did some change but there is no reset point for it and i have no idea how i'm gonna fix it. i did even made a empty and blank level and i did test it again but still zombie run - freeze - then all points go to start as you can see in my clip

**Comment by Real-Eragamer** [comment]
hey,
is there anything i can do?

**Comment by synchromesh62** [comment]
> hey, is there anything i can do?

I was hoping a verify would sort it :(
Basically its playing the initial run animation ( like in the Character Creator ) but is not actually moving by the look of it as it just resetting .. **One for Lee i think** .. Does it do it with all characters and attack settings like **Arms Out** or **Bad Back** or just that one particular **Dash** animation ?
Just so he has as much info as possible.

**Comment by Real-Eragamer** [comment]
> > hey, is there anything i can do?
> 
> I was hoping a verify would sort it :( Basically its playing the initial run animation ( like in the Character Creator ) but is not actually moving by the look of it as it just resetting .. **One for Lee i think** .. Does it do it with all characters and attack settings like **Arms Out** or **Bad Back** or just that one particular **Dash** animation ? Just so he has as much info as possible.

hey sorry for the delay of responding, only hunch back chase mode works fine and other get freeze

**Comment by Real-Eragamer** [comment]
hey i did delete scriptbank in gameguru max file and i all other folder about gameguru max in document exclude projectbank and then i did verfy and things got fixed but i lost the map design what i did about 3 month but at least i think i found out the problem was on document files.

**Comment by synchromesh62** [comment]
Never good to lose your work.
It always pays to back up your **GameGuruApps** folder ( which i do frequently ) then you could simply have copied your mapbank back or any other files you may have lost.

**Comment by Real-Eragamer** [comment]
> Never good to lose your work. It always pays to back up your **GameGuruApps** folder ( which i do frequently ) then you could simply have copied your mapbank back or any other files you may have lost.

I'll do this, thanks for advice. every failure i face , new thing i learn.
